posts/2023-06-16-smorgasbord-recap/index.qmd

```diff
@@ -0,0 +1,41 @@
+---
+title: "Bioconductor’s Participation in Galaxy Smorgasbord 2023: A Recap"
+description: |
+  Bioconductor made its first appearance at the Galaxy Smorgasbord this year. This post provides a summary of our successful participation.
+author:
+  - name: Maria Doyle
+    url: https://github.com/mblue9
+  - name: Alex Mahmoud
+    url: https://github.com/almahmoud
+date: 2023-06-16
+categories:
+  - events
+  - education
+image: "bioconductor-galaxy.png"
+---
+
+We're delighted to share that Bioconductor was part of the [Galaxy Smorgasbord](https://gallantries.github.io/video-library/events/smorgasbord3/) global online training for the first time this year. Over the week of May 22-26, we extended the power of R and Bioconductor to a wider, global audience.
```

Comment:
```suggestion
We're delighted to share that Bioconductor was part of the [Galaxy Smörgåsbord](https://gallantries.github.io/video-library/events/smorgasbord3/) global online training for the first time this year. Over the week of May 22-26, we extended the power of R and Bioconductor to a wider, global audience.
```

:grimacing: sorry for the terrible event name

Reply:
😄 thanks for fixing